typespec/UCP/resourceproviders.tsp

```diff
@@ -93,7 +93,7 @@ model ResourceTypeProperties {
   provisioningState?: ProvisioningState;
 
   @doc("The default api version for the resource type.")
-  defaultApiVersion: ApiVersionNameString;
+  defaultApiVersion?: ApiVersionNameString;
```

Comment:
Might have missed this in the design doc, but would making this optional require code changes? Why? Because now resource type doesn't have a default API version, and the code needs to go and get it from somewhere else or throw an error?

Reply:
For anyone else reviewing, this was a mistake in the initial .tsp. It was always supposed to be optional.

@ytimocin - you're making the assumption that anything reads this field. Nothing does yet.